### PR TITLE
Avatar,badge

### PR DESCRIPTION
**Pull request by ishapatell1**
Added Avatar and Badge component

### CODE REVIEW

File: Components/Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>UI-Avatar</title>
+    <link rel = "stylesheet" href = "/css/footer.css">
+    <link rel = "stylesheet" href = "/style.css">
+    <link rel = "stylesheet" href = "/css/navbar.css">
+    <link rel = "stylesheet" href = "/css/responsive.css">
+    <link rel = "icon" type = "image/x-icon" href = "/Images/favicon.png">
```
Comment (aqsaaqeel) [inline Components/Avatar/avatar.html:14]: You can put all of these CSS files in a folder. And import them in another index.css file

**Review by KanekarSnehal — APPROVED** [review]

File: Components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.avatar{
+    border-radius: 50%;
+    width:12rem;
+    margin:1rem;
+    aspect-ratio: 1/1;
+    padding: 2px;
+}
+.bahut-hi-chota{
```
Comment (iamharnad) [inline Components/Avatar/avatar.css:8]: Please use proper naming convention for the classes

File: Components/Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    width:2rem;
+}
+.bahut-chota{
```
Comment (iamharnad) [inline Components/Avatar/avatar.css:11]: Here as well

File: Components/Avatar/avatar.html (hunk 1 above)
Comment (ishapatell1) [reply Components/Avatar/avatar.html:14]: Sure, I will make the changes.

**Comment by iamharnad** [comment]
Please use proper naming conventions for the classes. Otherwise everything looks good

**Review by iamharnad — CHANGES_REQUESTED** [review]

**Pull request merged**